Dockerfile.openshift

```diff
@@ -0,0 +1,27 @@
+# This dockerfile is specific to building the network metrics daemon for OpenShift
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 as builder
+
+ENV GO111MODULE=on\
+      CGO_ENABLED=0
+
+WORKDIR /go/src/github.com/open-policy-agent/gatekeeper
+
+COPY pkg/ pkg/
+COPY third_party/ third_party/
+COPY vendor/ vendor/
```

Comment:
nit: can this be the first copy so we can cache it with working locally?

Reply:
I cloned the [original dockerfile](https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile), I think if we do that is easier to align if u/s change something.